### DISCUSSION TITLE
Error Parameter Not Found

### DISCUSSION THREAD

**Discussion by valannas** [builtbybel/FlyOOBE#40] [General]
Hi i am attempting to upgrade to Windows 11 on a 32bit computer running Windows 8.1. 

I was recommended to use your software to assist, however once I select version type etc, it pops up with an error stating "parameter not found UseBasicParsing" 

![20241221_103803](https://github.com/user-attachments/assets/6eba9d52-0be0-493c-9475-46efa8d5d85e)

I honestly don't know much about computers at all, and I'm trying to learn. Can anyone help out a very out of her depth mum? 😅 

I can't afford computer upgrades atm. 😭

**Comment by Belim** [comment]
hey, if the download from within the app doesn't work, you can simply download the image directly from the Microsoft site: https://www.microsoft.com/en-us/software-download/windows11 (scroll down on the page and start the download for Windows 11 Disk Image (ISO) for x64 devices). Afterward, you just need to click on the 'Mount and Run ISO' button in the Flyby11 app and select the image you previously downloaded. The rest will then follow in the step-by-step assistant. :) You'll manage it 😃
